package.json

```diff
@@ -29,6 +29,7 @@
     "_prepublish": "yarn small-build",
     "quick-build": "tsc --pretty",
     "small-build": "node scripts/build.js",
+    "build": "node scripts/build.js",
```

Comment:
This the same as `"small-build"` I would remove one of them

Reply:
I want to standardize to `build` but not to remove `small-build` in case someone still uses that for something.